modMcf/src/org/aion/mcf/config/CfgSync.java

```diff
@@ -38,13 +41,14 @@
     private int blocksQueueMax;
 
     private boolean showStatus;
+    private List<StatsType> showStatistics;
 
     private static int BLOCKS_QUEUE_MAX = 32;
 
     public CfgSync() {
         this.blocksQueueMax = BLOCKS_QUEUE_MAX;
-
         this.showStatus = false;
+        this.showStatistics = new ArrayList<>(Arrays.asList(StatsType.NONE));
```

Comment:
why not represent "NONE" with just an empty list?

Reply:
It's a possibility, but I'm not sure how user friendly that is. It definitely makes the comment in the config file longer. Up for debate though, if anyone else wants to pitch in.

Reply:
I like making NONE an explicit option, I think that is better for code maintenance / readability, and there's no (real) performance hit.